### PR TITLE
update partials to have leading _ and remove components from them

### PR DESCRIPTION
**Pull request by csestito**
## Description

Partial Builds were breaking due to some markdown not being imported as Markdown Partials.

Needed to follow the [partials convention](https://docusaurus.io/docs/markdown-features/react#importing-markdown) of a leading `_` before the file name. Also removed the props as they aren't needed for markdown with no input. 

## Screenshots (if appropriate)
Screen Shot 2022-11-15 at 9 40 44 AM

## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: products/ansible/docs/panos/tutorials/config-networking.mdx [hunk 1]
```diff
@@ -257,7 +272,7 @@ ok: [firewall] => {
 }
 
 PLAY RECAP *************************************************************************************************************************************************************************
-firewall                   : ok=3    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0 
+firewall                   : ok=3    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
 ```
 
-<ClosingNotes components={props.components} />
\ No newline at end of file
+<ClosingNotes components={props.components} />
```
Comment (sserrata) [inline products/ansible/docs/panos/tutorials/config-networking.mdx:278]: Not sure if `omponents={props.components}` is still needed here

File: products/ansible/docs/panos/tutorials/info-gather-sysinfo.mdx [hunk 2]
```diff
@@ -96,7 +101,7 @@ ok: [firewall] => {
 }
 
 PLAY RECAP *************************************************************************************************
-firewall           : ok=3    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
+firewall           : ok=3    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
 ```
 
-<ClosingNotes components={props.components} />
\ No newline at end of file
+<ClosingNotes components={props.components} />
```
Comment (sserrata) [inline products/ansible/docs/panos/tutorials/info-gather-sysinfo.mdx:107]: Not sure if `components={props.components}` is still needed

File: products/ansible/docs/panos/tutorials/operations-checkready.mdx [hunk 3]
```diff
@@ -96,7 +103,7 @@ ok: [firewall] => {
 }
 
 PLAY RECAP *************************************************************************************************
-firewall           : ok=2    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0    
+firewall           : ok=2    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
 ```
 
-<ClosingNotes components={props.components} />
\ No newline at end of file
+<ClosingNotes components={props.components} />
```
Comment (sserrata) [inline products/ansible/docs/panos/tutorials/operations-checkready.mdx:109]: Not sure if `components={props.components}` is still needed

File: products/ansible/docs/panos/tutorials/operations-upgrade-firewall.mdx [hunk 4]
```diff
@@ -114,9 +118,9 @@ ok: [firewall] => {
...
 PLAY RECAP ********************************************************************************************************************************
-firewall                   : ok=5    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0  
+firewall                   : ok=5    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
 ```
 
 6. Login to the PAN-OS GUI and confirm that the firewall is now running the target version of PAN-OS
 
-<ClosingNotes components={props.components} />
\ No newline at end of file
+<ClosingNotes components={props.components} />
```
Comment (sserrata) [inline products/ansible/docs/panos/tutorials/operations-upgrade-firewall.mdx:126]: Not sure if `components={props.components}` is still needed

**Review by sserrata — CHANGES_REQUESTED** [review]
There's a few `components={props.components}` left that could possibly be cleaned up

**Review by sserrata — APPROVED** [review]

**Review by jamesholland-uk — APPROVED** [review]

**Pull request merged**